yarn.lock

```diff
@@ -4160,7 +4160,7 @@ end-of-stream@^1.0.0, end-of-stream@^1.1.0, end-of-stream@^1.4.1, end-of-stream@
   dependencies:
     once "^1.4.0"
 
-enquirer@^2.3.0, enquirer@^2.3.6, enquirer@~2.3.6:
+enquirer@^2.3.0, enquirer@~2.3.6:
```

Comment:
Ouch, there are still references here... It looks like is a nested dependency of Nx

Reply:
Yes, it seems that cli will have to download this dependency anyway.